R/RcppExports.R

```diff
@@ -17,6 +17,10 @@ bds_Impl <- function(con_, securities, field, options_, overrides_, verbose, ide
     .Call(`_Rblpapi_bds_Impl`, con_, securities, field, options_, overrides_, verbose, identity_)
 }
 
+bds_Impl_Debug <- function(con_, securities, field, options_, overrides_, repeats, verbose, identity_) {
```

Comment:
As a PR to master, this is not needed.  Can you remove the function, and run `compileAttributes()` again to remove it here?